docs/download.md

```diff
@@ -52,27 +52,27 @@ do not have CUDA forward compatibility (for example, GeForce), CUDA 11.5 or late
 need to ensure the minimum driver (450.80.02) and CUDA toolkit are installed on each Spark node.
 
 ### Verify signature
-* Download the [RAPIDS Accelerator for Apache Spark 23.02.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.02.0/rapids-4-spark_2.12-23.02.0.jar)
-  and [RAPIDS Accelerator for Apache Spark 23.02.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.02.0/rapids-4-spark_2.12-23.02.0.jar.asc)
+* Download the [RAPIDS Accelerator for Apache Spark 23.04.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.04.0/rapids-4-spark_2.12-23.04.0.jar)
+  and [RAPIDS Accelerator for Apache Spark 23.04.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.04.0/rapids-4-spark_2.12-23.04.0.jar.asc)
 * Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
 * Import the public key: `gpg --import PUB_KEY`
-* Verify the signature: `gpg --verify rapids-4-spark_2.12-23.02.0.jar.asc rapids-4-spark_2.12-23.02.0.jar`
+* Verify the signature: `gpg --verify rapids-4-spark_2.12-23.04.0.jar.asc rapids-4-spark_2.12-23.04.0.jar`
 
 The output if signature verify:
 
 	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
 
 ### Release Notes
 New functionality and performance improvements for this release include:
-* Delta Lake MERGE/DELETE/UPDATE (experimental feature, can be enabled with a config flag)
-* Function `from_json`
-* Hive text table write 
-* Databricks 11.3 ML LTS support
-* Support batched full join to improve full join's performance
+* Optimize HashAggregate and Join operators with the new retry framework to avoid OOM errors and recover in some cases
```

Comment:
in addition to hash aggregate, this release already has the retry framework enabled in some capacity for: filter, gpu coalece, project, window aggregates. It will also be enabled to some extend in joins.

How about something like "Introduces OOM retry framework for automatic OOM handling in memory-intensive operators, such as: join, aggregates and windows, coalescing, projections and filters."

Join retry support has not been merged yet to 23.04.

Reply:
updated